Game.cs

```diff
@@ -8,18 +9,500 @@ internal class Game
+                                    currentRoom.removeItem("Shield");
+                                }
+                                else if (player.inventoryContents().Contains("Potion"))
+                                {
+                                    currentRoom.removeItem("Potion");
+                                }
+                                Console.WriteLine(currentRoom.getDescription());
+                                Console.WriteLine("1. Open the door");
+                                string answer = Console.ReadLine();
+                                while (playing == true && answer != "1")
+                                {
+                                    if (answer == "1")
+                                    {
+                                        Console.Clear();
+                                        Console.WriteLine("You open the door and walk through it.");
+                                        Console.WriteLine("You have reached the end of the dungeon!");
+                                        player.addScore(1000);
+                                        Console.WriteLine("Your score is: " + player.score);
+                                        playing = false;
+                                    }
+                                    else
+                                    {
+                                        Console.WriteLine("Invalid input.");
+                                    }
+                                }
+                            }
+                            else
+                            {
+
+                            }
+                            attack = rnd.Next(1, 10);
+                            if (attack == 1)
+                            {
+                                int difficulty = rnd.Next(1, 4);
+                                Monster monster = new Monster(monsterhealthsetting(difficulty), monsterdamagesetting(difficulty), difficulty);
+                                infight = true;
+                                Console.WriteLine("A monster appears in front of you!");
+                                while (infight)
+                                {
+                                    Console.WriteLine("What would you like to do?");
+                                    Console.WriteLine("1. Attack");
+                                    Console.WriteLine("2. Run");
+                                    Console.WriteLine("3. Use item");
+                                    Console.WriteLine("Monster health: {0}", monster.health);
+                                    Console.WriteLine("Your health: {0}", player.Health);
+                                    string fightinput = Console.ReadLine();
+                                    //begins the fight with the monster
+                                    switch (fightinput)
+                                    {
+                                        case "1":
```

Comment:
Again, ideally these cases should be split up into a group of methods so that the code is a bit more organised.